### PR TITLE
Code Documentation

### LINKED ISSUE #1: [QUALITY] Enriching the SDK code with SwiftDoc
# Problem
## What
Enrich SDK code with SwiftDoc (auto-generated documentation for Swift).

## Why
To help integrators better understand the SDK functionality.

## Who
Clients

# Solution
## How
- [x] Add comments on top of the public class in the java-doc format
- [x] Add comments on top of the public function in the java-doc format
- [x] Use swift-Doc utility to generate Documentation markup file [swift-doc](https://github.com/SwiftDocOrg/swift-doc)

### Supporting information
Sample Example of Documentation
``` swift
/// Text like this appears in "Description".
///
/// Leave a blank line to separate further text into paragraphs.
///
/// You can use bulleted lists (use `-`, `+` or `*`):
///
/// - Text can be _emphasised_
/// - Or **strong**
///
/// Or numbered lists:
///
/// 7. The numbers you use make no difference
/// 0. The list will still be ordered, starting from 1
/// 5. But be sensible and just use 1, 2, 3 etc…
///
/// ---
///
/// More Stuff
/// ==========
///
/// Code
/// ----
///
/// Use backticks for inline `code()`. Indentations of 4 spaces or more will create a code block, handy for example usage:
///
///     // Create an integer, and do nothing with it
///     let myInt = 42
///     doNothing(myInt)
///
///     // Also notice that code blocks scroll horizontally instead of wrapping.
///
/// Links & Images
/// --------------
///
/// Include [links](https://en.wikipedia.org/wiki/Hyperlink), and even images:
///
/// ![Swift Logo](/Users/Stuart/Downloads/swift.png "The logo for the Swift programming language")
///
/// - note: That "Note:" is written in bold.
/// - requires: A basic understanding of Markdown.
/// - seealso: `Error`, for a description of the errors that can be thrown.
///
/// - parameters:
///   - int: A pointless `Int` parameter.
///   - bool: This `Bool` isn't used, but its default value is `false` anyway…
/// - throws: A `BadLuck` error, if you're unlucky.
/// - returns: Nothing useful.
func doNothing(int: Int, bool: Bool = false) throws -> String {
    if unlucky { throw Error.BadLuck }
    return "Totally contrived."
}
```

### Expected change level
Minor.

### Dependencies
None.

# Acceptance criteria
- [x] As an SDK integrator, When I use any SDK function in my code, then XCode should show me proper documentation for that function.

# Documentation
N/A.

### PR DESCRIPTION
**Pull request by sameerdhulap**
# Description
## Issue
closes #1

## What
Added Comment on function call so developer know what exactly use of function

## How
Added swift format document on all function call in SDK

## Related PRs
N/A.

# Testing
## Automated tests
- [x] Unit Tests cover the change
- [x] Smoke Tests cover the change  

## Steps to test or reproduce
Please check the function code and check the code comment on 

# Deployment
## Strategy
- [x] This is a standard deployment

## Migrations

No.

### CODE REVIEW

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift [hunk 1]
```diff
@@ -659,8 +799,10 @@ open class LocationServiceCoreImpl: NSObject,LocationService,LocationServiceInte
         }
     }
     
-   
-    
+    /// Popilate Poi data properties
```
Comment (dippatra) [inline Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: please correct spelling

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift [hunk 2]
```diff
@@ -436,19 +526,26 @@ open class LocationServiceCoreImpl: NSObject,LocationService,LocationServiceInte
...
+    /// Capture error while monitoring region
+    /// - Parameters:
+    ///   - manager: location service
+    ///   - region: region info
+    ///   - error: Error info
     public func locationManager(_ manager: CLLocationManager, monitoringDidFailFor region: CLRegion?, withError error: Error) {
         print("WoosmapGeofencing Error : can't create geofence " + (region?.identifier ?? "") + error.localizedDescription)
     }
     
+    /// Remove olde poi for given region
```
Comment (dippatra) [inline Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: please correct spelling

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift [hunk 3]
```diff
@@ -92,7 +121,8 @@ open class LocationServiceCoreImpl: NSObject,LocationService,LocationServiceInte
             }
         }
     }
-
+    
+    /// Start mnitoring region
```
Comment (dippatra) [inline Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: correct spelling

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift (hunk 1 above)
Comment (sameerdhulap) [reply Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: resolved

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift (hunk 2 above)
Comment (sameerdhulap) [reply Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: resolved

**Review by dippatra — APPROVED** [review]

File: Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift (hunk 3 above)
Comment (sameerdhulap) [reply Sources/WoosmapGeofencing/Business Logic/LocationServiceCoreImpl.swift]: Corrected

**Review by dippatra — APPROVED** [review]

**Pull request merged**